observer/probers/tls/tls.go

```diff
@@ -90,6 +92,40 @@ func checkOCSP(cert, issuer *x509.Certificate, want int) (bool, error) {
 	return ocspRes.Status == want, nil
 }
 
+func checkCRL(cert, issuer *x509.Certificate, want int) (bool, error) {
```

Comment:
This function doesn't take a context and uses the default http client - could lead to issues if the requests hang for a long time. The caller has a context with a timeout so I think it would be good to pass it in here and use it.

I think especially in the context of boulder-observer (ie, monitoring for things going wrong), that's relatively important to have some sensible behaviour around misbehaving CRL servers

Reply:
The same is true of the checkOCSP function it is replicating, but yes I can make those improvements now instead of later.